### ISSUE TITLE
Vision Encoder Decoder Overfits

### ISSUE THREAD

**Issue by Serge9744** [NielsRogge/Transformers-Tutorials#295]
Hi @NielsRogge  , I have trained a Vision Encoder Decoder on French medical data , 2000 images on the training set , 330 for dev and test.
I augmented through random noie , blurring, contrasting, dowsnizing, upsizing,eroding , dilating , brightning to get roughly 20 000 images for training

I used Swin v2 base 256 (the images are roughly rectangular around 400pxl height and 1500 pxl wide) and Dr Bert small (but any decoder I used gave the same result)

The model overfits on the training data :

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/BENYAM~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/BENYAM~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

True labels :    ['Demande administrative', 'Renouvellement', 'echographie du coeur,   coronaire', 'EMPHYSME', '', 'amoxicilinne, prednisolone et ventoline',   'radios dos', 'Greffe Rénal / Annuloplastie Aortique'] |  
-- | --
predictions :  ['Demande administrative',   'Renouvellement', 'echographie du coeur, coronaire', 'EMPHYSME', '',   'amoxicilinne, prednisolone et ventoline', 'radios dos', 'Greffe Rénal /   Annuloplastie Aortique'] |  
True   labels :  ['TENSiON', 'Dépression   nerveuse', 'Prise de sang de contrôle', 'cesarienne', 'Rhumatismes -   athrose', 'Maladie Professionnelle tondinopathie 2011 année', 'PARACETAMOL',   'radio controle bilan sanguin']
predictions :  ['TENSiON', 'Dépression nerveuse', 'Prise   de sang de contrôle', 'cesarienne', 'Rhumatismes - athrose', 'Maladie   Professionnelle tondinopathie 2015', 'PARACETAMOL', 'radio controle bilan   sanguin']

</body>

</html>

but I keep having 0.66 CER for the dev set

My learning rate is 2e-5 .

Do you have a general recommandation apart from getting more images to decrease overfitting please? I do not know how can I tackle this issue

Best regards
